docs/source/endpoints/controlpanels.md

```diff
@@ -195,3 +195,206 @@ Response:
 ```{literalinclude} ../../../src/plone/restapi/tests/http-examples/controlpanels_delete_dexterity_item.resp
 :language: http
 ```
+
+### Content rules
+
+`@controlpanels/content-rules` is a custom control panel endpoint that will allow you to add, remove, and configure available {ref}`rules`.
+
+| Verb     | URL                                         | Action                                    |
+| -------- | ------------------------------------------- | ----------------------------------------- |
+| `GET`    | `/@controlpanels/content-rules`             | List configurable content rules           |
+| `POST`   | `/@controlpanels/content-rules`             | Creates a new content rule                |
+| `GET`    | `/@controlpanels/content-rules/{rule-id}`   | Get the current state of the content rule |
+| `PATCH`  | `/@controlpanels/content-rules/{rule-id}`   | Update the content rule details           |
+| `DELETE` | `/@controlpanels/content-rules/{rule-id}`   | Remove the content rule                   |
+
+
+#### Listing Content Rules
+
+To list the available content rules, send a `GET` request to `@controlpanels/content-rules`
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/controlpanels_get_contentrules.req
+```
+
+Response:
+
+```{literalinclude} ../../src/plone/restapi/tests/http-examples/controlpanels_get_contentrules.resp
+:language: http
+```
+
+The following fields are returned:
+
+- `@id`: hypermedia link to the rule
+- `id`: actual id of the content rule
+- `assigned`: rule assigned or not
+- `title`: title of the rule
+- `description`: rule description
+- `enabled`: rule is enabled or not
+- `trigger`: triggering event
+- `conditions`: conditions before triggering the rule
+- `actions`: actions to take place
+
+
+#### Creating a new Content rule with `POST`
+
+To create a new content rule, send a `POST` request to the `/@controlpanels/content-rules` endpoint:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/controlpanels_post_rule.req
+```
+
+Response:
+
+```{literalinclude} ../../src/plone/restapi/tests/http-examples/controlpanels_post_rule.resp
```

Comment:
@valipod Here I think there is a missing `../` As the response is not included within the final docs build. See screenshot:

![Control-Panels-—-plone-restapi-v8-24-2-dev0](https://user-images.githubusercontent.com/323385/191826024-f28d9ed3-d97b-4845-a0b6-4f5e20367c16.png)